.github/workflows/prepare-release.yml

```
@@ -81,12 +81,13 @@ jobs:
           export GITHUB_TOKEN=$( \
             curl --request GET --url ${{ secrets.VAULT_URL}} --header "Authorization: JWT ${{ secrets.VAULT_JWT }}" | jq -r .token \
           )
+          CURRENT_VERSION=$(cat package.json | jq .version | xargs)
```

I don't see the reason for using `xargs` here? 
```suggestion
          CURRENT_VERSION=$(cat package.json | jq -r .version)
```

---

to cut of the double quotes, will test that how it works without it